open_dread_rando/files/schema.json

```diff
@@ -313,9 +313,19 @@
                                     "type": "boolean",
                                     "default": false,
                                     "description": "Enables an in-game death counter shown in the player's HUD"
+                                },
+                                "room_ids": {
+                                    "type": "string",
+                                    "default": "NEVER",
+                                    "description": "The type of room name shown on GUI"
                                 }
                             },
                             "additionalProperties": false
+                        },
+                        "room_dict": {
+                            "type": "object",
+                            "description": "A dictionary of dictionaries mapping scenario and collision camera to room name in the database",
```

Comment:
Let's be precise in the schema!
This should be a dict of scenario name -> (dict of camera name -> area name), right?

```json
                "camera_names_dict": {
                    "type": "object",
                    "description": "...",
                    "additionalProperties": false,
                    "patternProperties": {
                        ".*": {
                            "type": "object",
                            "additionalProperties": false,
                            "patternProperties": {".*": {"type": "string"}}
                        }
                    }
                }
```

You should also make this key required, or change the python code to handle it being missing.

Reply:
if it's empty, the python side just passes an empty table to lua which would use collision cameras as a fallback. not sure if this is okay or not.

Reply:
If empty, sure. And that's what you passed so whatever.
But what if the key is _missing_?